app/js/components/searchDrug/View.jsx

```diff
@@ -1,53 +1,81 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+const showLoading = loading => (loading && <span> Searching...</span>);
+
+const showError = error => (
+  error && <span className="preformatted-note">{error.data.message}</span>
+);
+
 const View = ({
-  searchDrug, results, loading, error,
+  searchDrug, results, loading, error, focused,
+  onOptionSelected,
+  onChange, value,
 }) => {
   const renderResults = () => (
-    results && results.length > 0 &&
-    <datalist id="drugs">
+    results.length > 0 && focused &&
+    <div className="options-container">
       {results.map(result => (
-        <option value={result.display} key={result.uuid} />
+        <p
+          role="button"
+          className="option"
+          key={result.uuid}
+          onClick={(event) => {
+            onOptionSelected(result.uuid, result.display);
+          }}
+        >{result.display}
+        </p>
+
       ))}
-    </datalist>
+    </div>
   );
 
-  const showLoading = () => (loading ? ' Searching...' : '');
-
-  const showError = () => (error ? error.data.message : '');
+  const noDrugFound = () => (
+    !loading && focused && results.length === 0 &&
+    <span className="preformatted-note">
+      <span className="emoji-sorry" role="img" aria-label="search"> 🔍 </span>
+      Please refine your search
```

Comment:
Could this message be more descriptive?

Reply:
I didn't want to use `Drug not found` because the logic here is to show a generic message that notifies the user when his/her search term is less than 3 characters and when there is no result of the search, so I felt that telling the user to `refine his search` will capture both scenario. I am however open to better options. Thanks